### DISCUSSION TITLE
How to apply style to all control types?  (e.g. `<Style Selector="Control.mb2">`)

### DISCUSSION THREAD

**Discussion by bitbound** [AvaloniaUI/Avalonia#10318] [Q&A] [answered]
Is there any way to apply a style to all control types?  I'd expect it to look like the below, where available properties would be everything on the `Control` base class and could apply to all controls that inherit it.  However, this doesn't seem to work.

I tried searching, but couldn't find any info on it, or even anyone asking about it.  Seems like it'd be a common request, so maybe I'm missing something.

Thanks!

```
<Style Selector="Control.mb2">
	<Setter Property="Margin" Value="0,0,0,8" />
</Style>
```

**Comment by jp2masa** [comment, marked as answer]
Did you try `:is(Control)`? https://docs.avaloniaui.net/docs/styling/selectors#is

**Reply by bitbound** [reply]
Yup, I missed that.  `:is(Control).mb2` did the trick.  Thank you!
